exporters/committime/collector_base.py

```diff
@@ -224,6 +209,72 @@ def get_metric_from_build(self, build, app, namespace, repo_url):
             )
             logging.debug(e, exc_info=True)
             return None
+        except Exception as e:
```

Comment:
I am not always keen on catching Exception, but in this case I think it's ok'ish...

Reply:
That's how it already was. I just wanted to differentiate an AttributeError from any other Exception, because we already treated them the same way.

Reply:
Ok, no change in the logic then.